info/diktat-kotlin-coding-style-guide-en.md

```diff
@@ -1769,8 +1769,8 @@ class YourClass(var name: String) {
 }
 ```
 
-Also - init block was not added to Kotlin to help you simply initialize your properties it is needed for more conplex tasks. 
-So if `init` block contains only assignments of variables - move it directly to properties so they will be correctly initialized near the declaration.
+Also - init block was not added to Kotlin to help you simply initialize your properties it is needed for more complex tasks. 
+So if `init` block contains only assignments of variables - move it directly to properties, so they will be correctly initialized near the declaration.
```

Comment:
And what about this? Should we consider this logic it rule?

Reply:
Yes, I'm doing it right now.